evalml/automl/utils.py

```diff
@@ -77,6 +80,11 @@ def make_data_splitter(
         sklearn.model_selection.BaseCrossValidator: Data splitting method.
     """
     random_seed = random_seed
+    if n_splits is None:
+        if X.shape[0] < _SMALL_DATA_ROW_THRESHOLD and not is_time_series(problem_type):
+            n_splits = _SMALL_DATA_CV_FOLDS
+        else:
+            n_splits = 3
```

Comment:
Perhaps we could just leave the default to be 3 and reduce the conditional to just the if branch?  Functionally, it should be the same, yes?